builds/moment-timezone-with-data.min.js

```diff
@@ -1,7 +1 @@
-//! moment-timezone.js
-//! version : 0.5.13
-//! Copyright (c) JS Foundation and other contributors
-//! license : MIT
-//! github.com/moment/moment-timezone
```

Comment:
can you keep the header? you can find the latest header in the og version of with-data.min.json

Reply:
Added!